dm_control/_render/pyopengl/egl_renderer.py

```diff
@@ -46,7 +46,10 @@
 
 def create_initialized_headless_egl_display():
   """Creates an initialized EGL display directly on a device."""
-  for device in EGL.eglQueryDevicesEXT():
+  devices = EGL.eglQueryDevicesEXT()
```

Comment:
```suggestion  
  all_devices = EGL.eglQueryDevicesEXT()
  selected_device = os.environ.get("EGL_DEVICE_ID", None)
  if selected_device is None:
    candidates = all_devices
  else:
    device_idx = int(selected_device)
    if not 0 <= device_idx < len(all_devices):
      raise RuntimeError(
          f"EGL_DEVICE_ID must be an integer between 0 and "
          f"{len(all_devices) - 1} (inclusive), got {device_idx}.")
    candidates = all_devices[device_idx: device_idx + 1]
  for device in candidate_devices:
```

Reply:
Could you please document this environment variable [in the "Rendering" section of `README.md`](https://github.com/deepmind/dm_control/blob/master/README.md#rendering)?

Reply:
> Could you please document this environment variable [in the "Rendering" section of `README.md`](https://github.com/deepmind/dm_control/blob/master/README.md#rendering)?

done